cgi-bin/LJ/User/Account.pm

```diff
@@ -12,8 +12,11 @@
 # part of this distribution.
 
 package LJ::User;
+
 use strict;
-no warnings 'uninitialized';
```

Comment:
We may get a bunch of new warnings in dev server environment by taking this out, on code we inherited from LJ.